Update class.zoom-social-icons-widget.php

BMOv2600 opened the pull request:
Adds `class="zoom-social_icons-link"` to anchor links to increase customization possibilities.

BMOv2600 committed:
Update class.zoom-social-icons-widget.php

Adds `class="zoom-social_icons-link"` to anchor links to increase customization possibilities.

nvartolomei commented:
We're styling it via `.zoom-social_icons-list__item > a`, but indeed we can add another class to make things easier.

Just to stick to the conventions we are using for other classes, can you rename the class to `zoom-social_icons-list__link`, please?

BMOv2600 committed:
Update class.zoom-social-icons-widget.php

Renames anchor link class to "zoom-social_icons-list__link" to follow plugin naming conventions.
